README.adoc

```diff
@@ -23,25 +23,26 @@
 :page-seo-description: A getting started tutorial and an example on how to enable distributed tracing in Java microservices to easily trace request flows that span multiple resources by using MicroProfile Telemetry and Jaeger distributed tracing system.​
 :source-highlighter: prettify
 :guide-author: Open Liberty
-= Enabling distributed tracing in microservices using OpenTelemetry observability framework 
+
+= Enabling distributed tracing in microservices by using the OpenTelemetry observability framework 
```

Comment:
I think we need a shorter title- also do we want to call out OT or Microprofile? Or Jaeger?
```suggestion
= Enabling distributed tracing in microservices with OpenTelemetry and Jaeger
```

Reply:
@gkwan-ibm are there plans to deprecate the two existing guides that use MP OpenTracing? I believe the feature is now stabilized 

https://openliberty.io/guides/microprofile-opentracing.html
https://openliberty.io/guides/microprofile-opentracing-jaeger.html